addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/OutputPanel.java

```diff
@@ -1,256 +0,0 @@
-/*
- * Zed Attack Proxy (ZAP) and its related class files.
- *
- * ZAP is an HTTP/HTTPS proxy for assessing web application security.
- *
- * Copyright 2012 The ZAP Development Team
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.zaproxy.zap.extension.scripts;
-
-import java.awt.BorderLayout;
-import java.awt.EventQueue;
-import javax.script.ScriptException;
-import javax.swing.ImageIcon;
-import javax.swing.JButton;
-import javax.swing.JPanel;
-import javax.swing.JScrollPane;
-import javax.swing.JToolBar;
-import javax.swing.text.DefaultCaret;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.parosproxy.paros.Constant;
-import org.parosproxy.paros.extension.AbstractPanel;
-import org.zaproxy.zap.utils.DisplayUtils;
-import org.zaproxy.zap.utils.FontUtils;
-import org.zaproxy.zap.utils.ZapTextArea;
-import org.zaproxy.zap.view.ZapToggleButton;
-
-@SuppressWarnings("serial")
-public class OutputPanel extends AbstractPanel {
-
-    private static final long serialVersionUID = -947074835463140074L;
-    private static final Logger LOGGER = LogManager.getLogger(OutputPanel.class);
-
-    private static final ImageIcon CLEAR_ICON =
-            new ImageIcon(
-                    OutputPanel.class.getResource(
-                            "/org/zaproxy/zap/extension/scripts/resources/icons/broom.png"));
```

Comment:
I think this icon can be removed now. (Same for `ui-scroll-pane.png` and `ui-scroll-lock-pane.png`).